helm/mds/templates/cronjobsource.yaml

```diff
@@ -0,0 +1,17 @@
+{{- range $key, $processor := .Values.processors }}
```

Comment:
I think `$name` would be more meaningful than `$key` here - that's how it's set in develop

Reply:
agree on picking a naming convention (name is fine/preferred) ... we could/should apply this across the templates consistently (eg ingress.yaml) ... can help as as needed @evanxadkins